tests/integration/tests/test-home-page.js

```diff
@@ -0,0 +1,76 @@
+"use strict";
+
+const assert = require("assert");
+const HomePage = require("../pages/desktop/home.page");
+const NavBar = require("../regions/navbar.region");
+const UserDashboardPage = require("../pages/desktop/dashboard.page");
+
+describe("Firefox Monitor homepage", function() {
+  this.retries(2);
+
+  beforeEach(function() {
+    browser.url("/");
+  });
+
+  it("should load the latest breach card", function() {
+    const homePage = HomePage;
+
+    homePage.waitForPageToLoad();
+    assert(
```

Comment:
@jrbenny35 This seems to be the only instance of `assert()` I can find. Should this use chai's `expect()`?

Reply:
Good find!